std/string.d

```diff
@@ -193,12 +193,13 @@ public import std.uni : icmp, toLower, toLowerInPlace, toUpper, toUpperInPlace;
 public import std.format : format, sformat;
 import std.typecons : Flag;
 
-import std.meta;
-import std.range.primitives;
-import std.traits;
+import std.meta;// AliasSeq, staticIndexOf
+import std.range.primitives;// ElementEncodingType, ElementType, hasLength, hasSlicing, isBidirectionalRange, isForwardRange, isInputRange, isOutputRange, isRandomAccessRange
+import std.traits;// isConvertibleToString, isNarrowString, isSomeChar, isSomeString, StringTypeOf, Unqual
 
```

Comment:
That, and there's no space between `;` and `//`, and the comments should be aligned if possible.